daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -633,6 +634,10 @@ func (d *Driver) Remove(id string) error {
 	d.updateQuotaStatus()
 
 	if err := subvolDelete(d.subvolumesDir(), id, d.quotaEnabled); err != nil {
+		if sys.RunningInUserNS() && strings.Contains(err.Error(), "operation not permitted") {
```

Comment:
Would we be able to get an error-type instead of string matching?

Reply:
Doesn't seem straightforward https://github.com/moby/moby/blob/8e29dd734d20ddf503fb974190fac42cca71aa42/daemon/graphdriver/btrfs/btrfs.go#L267

Reply:
Isn't it coming from https://github.com/keybase/client/blob/master/go/vendor/golang.org/x/sys/unix/zerrors_linux_amd64.go#L2921 ?

Was thinking something like `errors.Is(err, os.ErrPermission)`

Reply:
That requires replacing `%v` with `%w`